delivery_free_fee_removal/tests/test_delivery_free_fee_removal.py

```diff
@@ -60,6 +63,10 @@ def test_delivery_free_fee_removal_with_fee(self):
                 }
             ],
         )
+        res = self.report_obj._get_report_from_name(
```

Comment:
Why do you need to test the report? The same question for the next ones.

Reply:
The purpose of testing the report is: improve the coverage and check that the delivery line is indeed displayed or not (depending on the use case).

Reply:
OK, I see now that there's a modification on the sales report.